### PR TITLE
Fix link to foresee asset

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Currently, the path to `foresee-trigger.js` does not include `assets/javascript`, therefore the network returns a 404. This issue was introduced as a result of moving from using `require` to load external scripts, to using `loadScript` (#15582)

## What is the value of this and can you measure success?

Correct path to foresee will ensure that lucky users get to see the foresee survey modal 🎉 

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE?

No

### CODE REVIEW

File: common/app/templates/javaScriptConfig.scala.js [hunk 1]
```diff
@@ -42,7 +42,7 @@
             "timingEvents": []
         },
         "libs": {
-            "foresee": "vendor/foresee/20150703/foresee-trigger.js",
+            "foresee": "/assets/javascripts/vendor/foresee/20150703/foresee-trigger.js",
```
Comment (sndrs) [inline common/app/templates/javaScriptConfig.scala.js]: should this be `@Static("javascripts/vendor/foresee/20150703/foresee-trigger.js")`?

**Comment by sndrs** [comment]
nice one, good catch. 👍 aside from comment

**Comment by stephanfowler** [comment]
What makes this resolve against https://assets.guim.co.uk - as opposed to against https://www.theguardian.com which is (part of?) the current problem?

**Comment by SiAdcock** [comment]
@sndrs the foresee files are not added to the asset hash. `foresee-trigger.js` loads `foresee-surveydef.js` from [a path hardcoded in the JavaScript](https://github.com/guardian/frontend/blob/master/static/vendor/javascripts/foresee/20150703/foresee-trigger.js#L10). So while we could potentially hash `foresee-trigger.js`, the rest would have to be served unhashed. I think that would make the behaviour more confusing.

**Comment by sndrs** [comment]
fair enough!

**Review by sndrs — APPROVED** [review]

**Comment by SiAdcock** [comment]
@stephanfowler I think you're right, since `foresee-trigger` is hosted on assets.guim.co.uk, we'll need to use `Static()` to resolve it. I've made the change, accordingly.

@sndrs can you give this another quick once-over?

**Review by sndrs — APPROVED** [review]

**Pull request merged**